src/CommunityToolkit.Maui.Core/CommunityToolkit.Maui.Core.csproj

```diff
@@ -81,4 +81,8 @@
     <None Include="..\..\build\nuget.png" PackagePath="icon.png" Pack="true" />
   </ItemGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="CommunityToolkit.Maui" />
```

Comment:
Why do we have internal on `Maui.Core`? 
@brminnick I remember that you mentioned that we shouldn't have `internals` on `Maui.Core`

Reply:
Good catch! Yes, let's remove this. 

If something in `CommunityToolkit.Maui.Core` is needed in `CommunityToolkit.Maui`, let's ensure it's available to everyone. If need be, we can always use `[EditorBrowsable(EditorBrowsableState.Never)]` like the `Microsoft.Maui` repo does.